ocamlformat-help.txt

```diff
@@ -331,6 +331,9 @@ OPTIONS (CODE FORMATTING STYLE)
            parentheses around tuples patterns. The default value is
            multi-line-only.
 
+       --parse-code-comments
+           Parse and format code in comments.
+
```

Comment:
why adding yet-an-other option for this? Can't just it be the default while parsing comments?

Reply:
I guess you are right, I will enable it by default.